variants/Cargo.toml

```
@@ -2,7 +2,9 @@
 members = [
     "aws-dev",
     "aws-ecs-1",
+    "aws-ecs-1.1",
     "aws-ecs-1-nvidia",
+    "aws-ecs-1.1-nvidia",
```

nit. While neither this, nor the above, is quite alphabetical, this might be more consistent with the rest:
```suggestion
    "aws-ecs-1",
    "aws-ecs-1-nvidia",
    "aws-ecs-1.1",
    "aws-ecs-1.1-nvidia",
```
Same goes for the README.md.